frappe/database/query.py

```diff
@@ -98,7 +98,7 @@ def apply_filters(
 		self,
 		filters: dict[str, str | int] | str | int | list[list | str | int] | None = None,
 	):
-		if not filters:
```

Comment:
What does empty string do here? 

`select name where name == ""`

That doesn't make any sense to me. PK is always set in framework.

Reply:
This is an older version of this line. It is now: `if filters is None`
https://github.com/frappe/frappe/pull/19405/files#diff-d578653ff2ad9f874d89ebb450b11619cc7771b9ce08dd03de2da25abc29cbc8R101